app/src/main/java/org/simple/clinic/home/patients/PatientsEffect.kt

```diff
@@ -2,12 +2,16 @@ package org.simple.clinic.home.patients
 
```

Comment:
There is a lot happening in this commit. We have removed `refreshApprovalStatusOnStart` and we are migrating displaying notifications. Can we try and split this commit ?

Reply:
In this commit, I migrated displaying of the "Waiting for approval" message only and it ended up letting us delete `refreshApprovalStatusOnStart` in the same PR.

Do you think I should move deleting the  `refreshApprovalStatusOnStart` message to another commit?

Reply:
Yea that would help. It's really difficult to review this PR as is because of the complexity of the migration.

Reply:
Done.